tests/parametric/test_otel_metrics.py

```diff
@@ -0,0 +1,2445 @@
+
+
+@features.otel_metrics_api
+@scenarios.parametric
+class Test_Otel_Metrics_Configuration_Metric_Export_Interval:
+    @pytest.mark.parametrize(
+        "library_env",
+        [
+            {
+                "DD_METRICS_OTEL_ENABLED": "true",
+                "CORECLR_ENABLE_PROFILING": "1",
+                "DD_TRACE_DEBUG": None,
+                "DD_TELEMETRY_HEARTBEAT_INTERVAL": "0.1",
+            },
+        ],
+    )
+    def test_default_interval(self, test_agent, test_library, library_env):
+        """SDK uses default interval when no interval env vars are set."""
+        name = "test_default_interval"
+
+        with test_library as t:
+            generate_default_counter_data_point(t, name)
+
+        metrics = test_agent.wait_for_num_otlp_metrics(num=1)
+        scope_metrics = metrics[0]["resource_metrics"][0]["scope_metrics"]
+        assert scope_metrics is not None
+
+        # Wait for telemetry configurations and verify the timeout has the default value of 10s
+        configurations_by_name = test_agent.wait_for_telemetry_configurations()
+
+        # Find default configurations (since no env vars are set, these should have default origin)
+        metrics_export_interval = test_agent.get_telemetry_config_by_origin(
+            configurations_by_name, "OTEL_METRIC_EXPORT_INTERVAL", "default", fallback_to_first=True
+        )
+
+        assert metrics_export_interval is not None, "OTEL_METRIC_EXPORT_INTERVAL should be set"
+
+        assert (
+            metrics_export_interval.get("value") == 10000
+        ), f"OTEL_METRIC_EXPORT_INTERVAL should be 10000, metrics_export_interval: {metrics_export_interval}"
+
+
+@features.otel_metrics_api
+@scenarios.parametric
+class Test_Otel_Metrics_Configuration_Metric_Export_Timeout:
+    @pytest.mark.parametrize(
+        "library_env",
+        [
+            {
+                "DD_METRICS_OTEL_ENABLED": "true",
+                "CORECLR_ENABLE_PROFILING": "1",
+                "DD_TRACE_DEBUG": None,
+                "DD_TELEMETRY_HEARTBEAT_INTERVAL": "0.1",
+            },
+        ],
+    )
+    def test_default_timeout(self, test_agent, test_library, library_env):
+        """SDK uses a non-standard default timeout when no timeout env vars are set."""
+        name = "test_default_timeout"
+
+        with test_library as t:
+            generate_default_counter_data_point(t, name)
+
+        metrics = test_agent.wait_for_num_otlp_metrics(num=1)
+        scope_metrics = metrics[0]["resource_metrics"][0]["scope_metrics"]
+        assert scope_metrics is not None
+
+        # Wait for telemetry configurations and verify the timeout has the default value of 10s
+        configurations_by_name = test_agent.wait_for_telemetry_configurations()
+
+        # Find default configurations (since no env vars are set, these should have default origin)
+        metrics_export_timeout = test_agent.get_telemetry_config_by_origin(
```

Comment:
This pattern is a bit odd. It would be nice if `get_telemetry_config_by_origin`, called `wait_for_telemetry_configurations()`, but that's beyond the scope of this PR